mermaid.go

```diff
@@ -19,7 +19,7 @@ var (
 	MermaidShapeStadium      = shape{"([", "])"}
 	MermaidShapeSubroutine   = shape{"[[", "]]"}
 	MermaidShapeCylinder     = shape{"[(", ")]"}
-	MermaidShapeCirle        = shape{"((", "))"}
```

Comment:
this is however a  breaking change. Maybe we should add a new and mark the old as deprecated instead.

Reply:
Good catch! Fixed.